### PR TITLE
FE Lighthouse dashboard

### PR DESCRIPTION
**Pull request by oseasmoran73**
## Description

[Ticket](https://app.zenhub.com/workspaces/vsp---frontend-tools-5fc9325744944e0015ed1861/issues/department-of-veterans-affairs/va.gov-team/28386) associated with PR

This PR creates a dashboard for lighthouse report that is generated from daily lighthouse scan in [vets-website](https://github.com/department-of-veterans-affairs/vets-website/actions/workflows/daily-lighthouse-scan.yml)

## Testing done

Locally (see screenshot below)

## Screenshots

image

### CODE REVIEW

File: packages/documentation/src/pages/frontend-support-dashboard/lighthouse-performance-report/app.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <Layout location={location}>
+      <div className="vads-l-row site-l-wrapper">
+        <Sidebar location={location} />
+        <div
+          id="main-content-coverage"
+          className="dash-coverage-table site-c-content__content docSearch-content vads-l-col--12"
+        >
+          <h1>Frontend Support Dashboard</h1>
+          <h2>Lighthouse Performance Report</h2>
+          <Table data={coverageReportData} fields={coverageReportFields} />
```
Comment (timwright12) [inline packages/documentation/src/pages/frontend-support-dashboard/lighthouse-performance-report/app.jsx:55]: Just requesting a UI change here from table to UL with links so we're not displaying the URL

**Review by timwright12 — CHANGES_REQUESTED** [review]

File: packages/documentation/src/pages/frontend-support-dashboard/lighthouse-performance-report/app.jsx (hunk 1 above)
Comment (oseasmoran73) [reply packages/documentation/src/pages/frontend-support-dashboard/lighthouse-performance-report/app.jsx]: Pushed change to use ul/li and no longer `table`. Updated screenshot to reflect new look

File: packages/documentation/src/pages/frontend-support-dashboard/lighthouse-performance-report/lighthouse-report.json [hunk 2]
```diff
@@ -0,0 +1,210 @@
+{
+  "homepage": {
+    "path": "/",
+    "s3": "https://vetsgov-website-builds-s3-upload.s3-us-gov-west-1.amazonaws.com/lighthouse/homepage.html"
+  },
+  "/auth/login/callback": {
+    "path": "/auth/login/callback",
+    "s3": "https://vetsgov-website-builds-s3-upload.s3-us-gov-west-1.amazonaws.com/lighthouse/auth/login/callback.html"
```
Comment (timwright12) [inline packages/documentation/src/pages/frontend-support-dashboard/lighthouse-performance-report/lighthouse-report.json:8]: @oseasmoran73 can we move this baseURL (`https://vetsgov-website-builds-s3-upload.s3-us-gov-west-1.amazonaws.com/lighthouse/`) into the jsx file? I'm thinking of the situation where we move the files so we don't have to update the entire file

Comment (oseasmoran73) [reply packages/documentation/src/pages/frontend-support-dashboard/lighthouse-performance-report/lighthouse-report.json]: Done!

**Review by timwright12 — APPROVED** [review]

**Pull request merged**